skeletons/constr_SEQUENCE.c

```diff
@@ -1835,7 +1835,7 @@ SEQUENCE_encode_aper(const asn_TYPE_descriptor_t *td,
 
 		/* Eliminate default values */
 		if(present && elm->default_value_cmp
-		        && elm->default_value_cmp(memb_ptr2) == 0)
+		        && elm->default_value_cmp(*memb_ptr2) == 0)
```

Comment:
Could you please explain why you're dereferencing the pointer here? And shouldn't it be checked for NULL?

Reply:
It is a pointer to a pointer and the default_value_cmp expectes a pointer to the value. Its the same in the UPER version.
A NULL check shouldn't be required unless the construction of a AFT_POINTER is broken (and the member is not really a pointer).